packages/nivo-pie/index.d.ts

```diff
@@ -0,0 +1,124 @@
+declare module '@nivo/pie' {
+    export class Pie extends React.Component<PieProps & Dimensions>{}
+    export class ResponsivePie extends React.Component<PieProps>{}
+
+    export type LegendItemDirection =
+    | 'left-to-right'
+    | 'right-to-left'
+    | 'top-to-bottom'
+    | 'bottom-to-top';
+
+    export type Anchor =
+        | 'top'
+        | 'top-right'
+        | 'right'
+        | 'bottom-right'
+        | 'bottom'
+        | 'bottom-left'
+        | 'left'
+        | 'top-left'
+        | 'center';
+
+    export type LegendDirection = 'row' | 'column';
+
+    export type Legend = {
+
```

Comment:
Please review the optional props. in Legend. Is this correct?

Reply:
Seems ok